From ee51521aecf0944d0b04445ad2ee17825a61457c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 02 Aug 2022 18:38:33 +0300
Subject: [PATCH] [SW-98239] erom: recovery: remove sp sram init

Gaudi2 ARC preboot_sp_sram_init() function is not defined and resolves
to weak definition that is just stub.
Therefore its callback is just misleading. Remove it.

Change-Id: I95d04b0d7a7fc2366de5dbd7c5e5f7ddc947aa99
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ext_rom/erom_recovery.c b/app/ext_rom/erom_recovery.c
index 70542c7..57dbc01 100644
--- a/app/ext_rom/erom_recovery.c
+++ b/app/ext_rom/erom_recovery.c
@@ -104,8 +104,6 @@
 	spi_flash_init();
 	spi_flash_prot_status();
 
-	preboot_sp_sram_init();
-
 	preboot_device_id_init();
 
 	preboot_security_init();
